src/cdk-experimental/menu/menu-item.ts

```diff
@@ -71,6 +63,12 @@ export class CdkMenuItem implements FocusableOption, FocusableElement, Toggler,
   }
   private _disabled = false;
 
+  /**
+   * The text used to locate this item during menu typeahead. If not specified,
+   * the `textContent` of the item will be used.
+   */
+  @Input() typeahead: string;
```

Comment:
That would mean there's no way to make the typeahead in the CDK version work with icons or other text that isn't supposed to be part of the typeahead. If you wanted to change it you'd have to extend the class and override the method

Reply:
I don't think that customizing the typeahead is that common. We could leave it out and it'll be straightforward if we get a feature request for it.

Reply:
IMO putting ligature icons inside menus is really common and we should have _some_ accommodation for it

Reply:
This typeahead feature isn't something most users know about, I don't think that we should overcomplicate it.

Reply:
I don't think it would be a very good experience to offer this feature (typeahead), but not have an API to configure it to actually work for your use case. As a developer trying to use this I would just find that very frustrating

Reply:
I'm really not convinced that people would care enough about the case where users press letter keys while a menu is open. I don't feel strongly enough to block the PR though.